packages/components/src/vue/components/morph.vue

```diff
@@ -59,13 +62,24 @@ export default {
     }
   },
   methods: {
+    inflectionListClass (lex) {
+      if (lex.getSelectedInflection()) {
+        if (lex.inflections.length === 1) {
+          return "dupe-infl-set"
+        } else {
+          return "super-infl-set"
+        }
+      } else {
+        return "full-infl-set"
+      }
```

Comment:
I am setting these classes here, rather than in the inflection-list component, because its' the duplication of the inflection-list component specified in the morph component that makes it necessary to distinguish between the two with classes. 

I am not sure if we will hide or show the full inflection set in the Alpheios Reader pages, so I want to make this granularly manageable via the css classes (eventually to be moved to data attributes)